python/matplotlib.ipynb

```diff
@@ -477,16 +481,43 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Seit der matplotlib 2.0.2 werden die Kappen der Fehlerbalken nicht mehr angezeigt, um das manuel hinzuzufügen wird\n",
```

Comment:
Ich finde caps auf den Fehlerbalken nicht so wichtig, das man sie hier erwähnen müsste. Wie sehen das die anderen?

Reply:
Also ich finde das eigentlich schöner.
Deshalb sollte man wenigsten zeigen wie es funktioniert, finde ich.

Reply:
Keep it simple!

Das ist eine Sache mehr, die wir zeigen und erklären müssen. Meiner Meinung nach sind diese Caps sogar störend. Es ist ein zusätzliches Element, was keinerlei Informationen trägt. Das sorgt nur für visual clutter.